components/ActivityMap/MapContainer.tsx

```
@@ -0,0 +1,199 @@
+        };
+
+        const subs = await DataLoader.loadRegions(viewportBounds);
+        const duration = (performance.now() - startTime).toFixed(2);
+
+        logger.debug(`[MapContainer] Loaded ${subs.length} regions (${duration}ms)`);
+        setregions(subs);
+    }, [map]);
+
+    // setup map event listeners with debouncing
+    useEffect(() => {
+        if (!map) return;
```

ESLint

[error] Expected { after 'if' condition.

(curly)